ts/components/session/LeftPaneContactSection.tsx

```diff
@@ -90,6 +85,8 @@ export class LeftPaneContactSection extends React.Component<Props, State> {
 
   public componentDidMount() {
     MainViewController.renderMessageView();
+
+    window.Whisper.events.on('calculatingPoW', this.closeOverlay);
   }
 
   public componentDidUpdate() {
```

Comment:
for readability sake, can we move `componentDidMount` and `componentDidUpdate` above `componentWillUnmount`?

Reply:
fixed